uk_bin_collection/uk_bin_collection/councils/RenfrewshireCouncil.py

```
@@ -17,275 +17,45 @@ class CouncilClass(AbstractGetBinDataClass):
 
-            # Try to accept cookies if the banner appears
-            try:
-                accept_button = WebDriverWait(driver, 10).until(
-                    EC.element_to_be_clickable((By.ID, "ccc-notify-accept"))
-                )
-                accept_button.click()
-                time.sleep(2)
-            except:
-                pass
+        # Make the GET request
+        response = requests.get(URI)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)